howto/manage-models.md

```diff
@@ -16,80 +16,107 @@ In many scenarios it is necessary to migrate models to and from an environment t
 
-First we must check that we have a cloud-credential for the cloud where the desired model is running.
-This is simply a pre-check performed when importing a model to ensure that the user has credentials for the cloud.
+It may not be possible to know all users that have have consumed offers when you wish to migrate a model, but 
+using [juju show-offer](https://documentation.ubuntu.com/juju/3.6/howto/manage-offers/#view-an-offers-details)
+will help to see all users that currently have access to an offer. This list should help determine which users
+to specify in the user mapping file.
 
-Check with the following,
+With a user mapping created, we can move onto the next step.
 
-```text
-juju switch jimm
-juju list-credentials --controller
+### 3. Validate cloud-credentials
+
+Next we must check that we have a valid cloud-credential for the incoming model.
+Run `juju show-model` and look for the `credential` field which should resemble the below:
+```yaml
+  credential:
+    name: lxd-creds
+    owner: admin
+    cloud: localhost
+    validity-check: valid
 ```
 
-If you do not see a cloud-credential for the desired cloud, you can add one by following the instructions in [managing cloud-credentials](https://juju.is/docs/juju/manage-credentials).
+The new model owner must have a cloud-credential with the same name and for the same cloud. 
 
-We then need the model UUID to import the model.
+Using the credential details above as an example - if model `admin/foo` is being migrated
+and the user mapping contains the row `admin: joe@canonical` then `joe@canonical` must have a 
+credential (`juju show-credentials --controller`) named `lxd-creds` for cloud `localhost` in JIMM.
+
+If you do not see a matching cloud-credential, you can add one by following the instructions in [managing cloud-credentials](https://juju.is/docs/juju/manage-credentials).
+
+### 3. Migrate desired models
+
+Once you have identified which models to migrate, created a user mapping file and validated that 
+the new owner has a valid cloud-credential, we can begin the process of model migration.
+
+We will assume a model called `admin/my-model` is currently hosted on a controller called `my-controller` and a controller
+called `workload-lxd` is connected to JIMM, where JIMM is known to the CLI as `jimm`.
+
+A user mapping file called `test-mapping.yaml` is created and placed in `~/snap/juju/common/` to avoid Snap permission issues.
 
 ```text
-MODEL_NAME="my-model"
-juju switch workload-lxd:$MODEL_NAME
-MODEL_UUID=$(juju show-model $MODEL_NAME --format yaml | yq .$MODEL_NAME.model-uuid)
-juju switch jimm-k8s
-# Replace <user-email> below with your email address
-juju import-model workload-lxd $MODEL_UUID --owner <username>
+juju switch my-controller:admin/my-model
+juju jaas migrate admin/my-model jimm --backing-controller=workload-lxd --user-mapping="/home/user/snap/juju/common/test-mapping.yaml"
```

Comment:
This command would ideally remove the `jaas` bit but we haven't yet made the changes to the Juju CLI. So I've kept the nested use of the jaas plugin until those changes are made.